libcni/api.go

```diff
@@ -165,33 +178,33 @@ func (c *CNIConfig) ensureExec() invoke.Exec {
 	return c.exec
 }
 
-func getResultCacheFilePath(netName string, rt *RuntimeConf) string {
+func (c *CNIConfig) getResultCacheFilePath(netName string, rt *RuntimeConf) string {
 	cacheDir := rt.CacheDir
```

Comment:
maybe we should use c.cacheDir instead of rt.CacheDir directly because rt.CacheDir is deprecated

Reply:
Done